spyder/plugins/configdialog.py

```diff
@@ -1090,6 +1090,15 @@ def setup_page(self):
     def apply_settings(self, options):
         self.set_option('selected', self.current_scheme)
         self.main.editor.apply_plugin_settings(['color_scheme_name'])
+        if self.main.ipyconsole is not None:
+            msg_title = "Restart Needed"
+            msg = _("In order to color the IPython console, Spyder needs to "
+                    "be restarted. Select Yes if you want to do it now or No "
+                    "if you will do it later manually.")

```

Comment:
Actually is not qtconsole that doesn't work but JupyterWidget. The function that doesn't work is: ```_syntax_style_changed()```